### DISCUSSION TITLE
Jira Basic authentication with passwords is deprecated

### DISCUSSION THREAD

**Discussion by berniemoran** [jertel/elastalert2#746] [Q&A] [answered]
No longer able to use username/password for Jira tickets.
Are there any plans to use the API token access method?

https://developer.atlassian.com/cloud/confluence/deprecation-notice-basic-auth/

**Comment by jertel** [comment, marked as answer]
Have you tried specifying the API token as the password?

**Reply by jertel** [reply]
FYI, #750 was recently merged to add direct support for API token authentication.
